frontend/rolecall/src/app/performance/performance-editor.component.html

```diff
@@ -202,12 +202,15 @@ <h3 class="input-title">Description</h3>
             </div>
             <div class="segment-list app-card">
               <div class="segment-container" *ngFor="let segment of state.step_3.segments">
-                <h3 class="title">{{segment['name']}}</h3>
+                <!-- <h3 class="title">{{segment['name']}}</h3> -->
```

Comment:
Please remove the commented out code.

Reply:
When I went back to the file to do this, my new code showed errors. The code compiled and worked, but it seems as if the VS Code plugin has problems with the new code (it seems to intermittent because it showed no errors when I made the changes).

Still, give this intermittent problem, I reverted to the old code.

Reply:
I would suggest to revert by using commit version and not check commented code in.